client/src/helpers/Errors.ts

```diff
@@ -0,0 +1,25 @@
+import { AxiosError } from 'axios';
+
+export default class Errors {
+  static handleRequestError(
```

Comment:
no need for this abstraction as it will be different for different requests, maybe you can abstract the `Something went wrong` message into a constant

Reply:
Yes, we discussed about it yesterday and you are completely right about it. but here without this abstraction, I need to write the same block of code three times. I need to discuss this issue with you. In the meantime let's ignore this issue.